doc/api.rst

```diff
@@ -40,126 +46,119 @@ Custom objects for storing extracted data.
 .. currentmodule:: lisc.data
 
 .. autosummary::
-  :toctree: generated/
+    :toctree: generated/
 
-  Term
-  Articles
-  ArticlesAll
-  MetaData
+    Term
```

Comment:
@ryanhammonds : this was still the case, and it's because the `data_object.rst` template needed to have the `:no-inherited-members:` option, which stops it grabbing inherited members from the NamedTuple. I think the reason it didn't get copied across could be that it got added in a different commit from the rest in the FOOOF repo. Anyway, since it was one little thing, to just finish this PR, I directly committed what was needed.